packages/web/server/api/routers/bridge-transfer.ts

```diff
@@ -334,10 +344,7 @@ export const bridgeTransferRouter = createTRPCRouter({
         throw new Error("Invalid bridge provider id: " + input.bridge);
       }
 
-      const supportedAssetFn = () => bridgeProvider.getSupportedAssets(input);
-
-      /** If the bridge takes longer than 10 seconds to respond, we should timeout that query. */
-      const supportedAssets = await timeout(supportedAssetFn, 10 * 1000)();
```

Comment:
Is this expected? Perhaps it's better to increase the timeout

Reply:
I just removed this to use the default 12 seconds timeout, since providers can be slow sometimes